docs/configuration-reference.md

```diff
@@ -138,6 +138,16 @@ key | value | description
 :--|:--|:--
 field | string | The name of the field to drop.
 
+### scrub_field
+
+The `scrub_field` processor will replace the specified field with a one-way hash before sending them to Honeycomb. This is useful for sanitizing sensitive information from events, while still allowing for inclusion and frequency analysis. May be specified multiple times.
+
+**Options:**
+
+key | value | description
+:--|:--|:--
+field | string | The name of the field to scrub.
+
```

Comment:
Can you add an example configuration snippet here as well? Similar to what we have for the `addtional_fields` processor. This is part of a larger effort where we need to have clearer docs on how to use these options.

Reply:
How about bd3578e ?

Reply:
How about bd3578e?